Dockerfile

```diff
@@ -32,4 +38,4 @@ WORKDIR /opt/local/src/CanarieAPI/canarieapi
 CMD env >> /etc/environment && \
     cron && \
     nginx && \
-    gunicorn -b 0.0.0.0:2000 --workers 1 --log-level=DEBUG --timeout 30 -k gevent wsgi
\ No newline at end of file
+    gunicorn -b 0.0.0.0:2000 --workers 1 --log-level=DEBUG --timeout 30 -k gevent canarieapi.wsgi
```

Comment:
While we are here could we swap gunicorn and nginx process so that the container logs keep the nginx ones. gunicorn doesn't yield any usefull logs anyway.
```
gunicorn -b 0.0.0.0:2000 --workers 1 --log-level=DEBUG --timeout 30 -k gevent canarieapi.wsgi --daemon && \
nginx -g "daemon off;"
```